src/option_types.hh

```diff
@@ -105,6 +106,8 @@ Vector<T, domain> option_from_strings(Meta::Type<Vector<T, domain>>, ConstArrayV
     return res;
 }
 
+bool option_add_from_strings(Vector<RangeAndString, MemoryDomain::Options>& opt, ConstArrayView<String> strs);
```

Comment:
I think this should be in highlighters.hh next to the other RangeAndString methods.